### PR TITLE
DBZ-173 Upgrades the Kafka artifact versions to 0.10.1.1

### PR DESCRIPTION
**Pull request by hchiorean**
The performance issue described [in this comment](https://issues.jboss.org/browse/DBZ-173?focusedCommentId=13350953&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13350953) was addressed by setting a low enough value for the new `max.poll.interval.ms` consumer property in the `DatabaseHistoryTest`. 

This seems perfectly acceptable since the test is running everything locally, however it's unclear to me atm. if in a real cluster the default value of `300 seconds` is ok or not. As mentioned in the JIRA issue, using this default value completely "destroys" the Kafka Consumer performance on my Windows machine

### CODE REVIEW

**Comment by hchiorean** [comment]
the current build failure seems to be again around the MySQL ITs, but it seems like more tests are failing than on previous builds.
I'm not sure if this is due to the upgrade or unrelated

**Comment by hchiorean** [comment]
I'll investigate this locally a bit more and see if the same failures occur

**Comment by rhauch** [comment]
@hchiorean, I've fixed the MySQL integration test issues on `master`, so you may want to rebase.

**Comment by hchiorean** [comment]
@rhauch: I've rebased on latest `master` and added an additional commit which changes some of the MySQL integration tests logic to be able to use `database.hostname` (system properties) as opposed to always hardcoding `localhost` (which doesn't work with Docker on certain OSes)

Now, I still get 2 types of failure for these ITs (both locally and on Travis)
1. `'source/pos' field in the record value is 2922711 but expected 2913420` 
2. Expected :'DROP TABLE IF EXISTS regression_test.dbz84_integer_types_table'
Actual   :'DROP TABLE IF EXISTS \`regression_test\`.\`dbz84_integer_types_table\`'

With (1) I have absolutely no clue what's going on, especially since the expected `source/pos` is hardcoded in `JSON` files and the rest of the `SourceRecord` content matches. It actually makes me wonder of the usefulness of these particular assertions since the tests seem extremely brittle. I doubt that upgrading the Kafka version has any influence on these types of failures.

With (2) I'm wondering how the tests have been working up until now: [this code](https://github.com/debezium/debezium/blob/master/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java#L295) clearly indicates that the MySQL connector `quotes` DDL identifiers and therefore the `Actual ouput` from the tests is correct. What I don't understand is why the `Expected output` doesn't use these quotes. I can only speculate that perhaps something changed (maybe a bug fix) somewhere in the Kafka Connect JSON converters, but you may have more info. 

In other words, (2) may be caused by the version change but I doubt (1) is.

**Comment by rhauch** [comment]
@hchiorean I think (2) has become a problem because of a change in the MySQL server, likely due to a bug fix that showed up recently. The connector has been quoting the names of the tables when it runs `SHOW CREATE TABLE <quotedTableName>`, but just the resulting DDL statement returned by the server started containing those same quotes. Prior MySQL versions stripped the quotes. I'll work on fixing these in a separate PR.

**Comment by rhauch** [comment]
@hchiorean I've made the changes in PR #169 and then merged that into `master`. Can you rebase again?

**Comment by hchiorean** [comment]
@rhauch: I've rebased locally but now I'm starting to get failures like:
```
Expected 1970-01-01T00:00:00-06:00 
Actual 1970-01-01T00:00:00+02:00
```
where the actual DB inserted record is a `TIMESTAMP` with the `1970-01-01T00:00:00` value.

obviously hardcoding TZ information in the expected results is not a good idea IMO, but fixing this requires a bit a of work since `VerifyRecord` has to be changed as well for certain types which have TZ information in order to make the correct assertions.

atm. I'm making the above changes...

**Comment by hchiorean** [comment]
@rhauch: it turns about the above issue is more complicated than I originally thought, because of the way the MySQL log reader behaves versus the JDBC driver (i.e. snapshot reader):

Given a MySQL `TIMESTAMP(2)` column (which is a `io.debezium.time.ZonedTimestamp`) with the value `2014-09-08 17:51:04.777` (inserted via DML)
1. the Binlog Reader treats this as a `OffsetDateTime` instance using the _system default TZ_ as timezone
2. the Snapshot Reader (aka. JDBC driver) instance returns an in-memory `java.sql.Timestamp` column at the _system default TZ_ which the `JdbcValueConverters` class (using `adaptive` behavior) converts to `UTC`. 

Considering that on my machine the TZ is `UTC+2`:

(1) produces `2014-09-08 17:51:04.77+02` while
(2) produces `2014-09-08 14:51:04.77Z` (due to UTC conversion plus DST on that particular date)

Given that both (1) and (2) are `io.debezium.time.ZonedTimestamp` and both correspond to the same DB column type, but at runtime different output values are produced depending on whether a snapshot is taking place or not, I'm not sure how this can be accounted for in the tests as far as the expected output goes.

These integration tests are turning out to be a PITA :)

**Comment by hchiorean** [comment]
@rhauch: from my previous rant, I'm thinking that there's at least one bug (non backwards compatible in terms of Kafka Records content) which should be fixed

**Comment by hchiorean** [comment]
I've opened https://issues.jboss.org/browse/DBZ-183 which has to be fixed first before I can continue investigating the integration tests.

**Comment by rhauch** [comment]
@hchiorean I just merged #170 for [DBZ-183](https://issues.jboss.org/browse/DBZ-183). Do you want to try rebasing again?

**Comment by hchiorean** [comment]
Rebased on latest `master` and added a new commit which 
a) makes sure no TZ information is present in the expected JSON files and 
b) fixes an additional problem where the expected files readers were not using UTF8 causing failures on my machine
